pulp_ansible/tests/functional/conftest.py

```diff
@@ -0,0 +1,118 @@
+import uuid
```

Comment:
To recap our matrix convo for historical purposes, we put them in conftest.py when they are not a pytest plugin (pulp_ansible currently is not). If it is a pytest plugin with the entrypoint style declaration we put it into `__init__.py` because that's how pytest likes it.

Reply:
Oh and also on Matrix we identified pulp_ansible should likely be a pytest plugin so that it's fixtures are available to galaxy_ng for test writing also. When that happens we'll likely move it back. I hadn't thought of that when I moved them.